rotkehlchen/tests/api/test_data_import.py

```diff
@@ -409,3 +412,80 @@ def test_data_import_custom_format(rotkehlchen_api_server, file_upload):
     assert result is True
     # And also assert data was imported succesfully
     assert_custom_cointracking(rotki)
+
+
+def test_associated_locations(user_data_dir, rotkehlchen_api_server):
```

Comment:
As I wrote before this is an integration test while it should have been a unit test.

What are you testing here? You are setting up an entire rotki instance, and making end to end api queries ... all to test a single database query.

This should be a unit test. Populate the DB and then check get_associated_location() returns them correctly. Also test location is wrong. It's nothing to do with data import.

This should be a db unit test.